dnplab/dnpNMR.py

```diff
@@ -95,20 +63,21 @@ def remove_offset(all_data, proc_parameters):
 
     Example::
 
-       proc_parameters = {}
-       proc_parameters['dim'] = 't2'
-       proc_parameters['offset_points'] = 10
-
-       workspace = dnplab.dnpNMR.remove_offset(workspace, proc_parameters)
+       workspace = dnplab.dnpNMR.remove_offset(workspace)
     """
 
     # Determine if data is dictionary or dnpdata object
     data, isDict = return_data(all_data)
 
-    requiredList = _default_remove_offset_parameters.keys()
-    proc_parameters = update_parameters(
-        proc_parameters, requiredList, _default_remove_offset_parameters
-    )
+    proc_parameters = {
+        "dim": dim,
+        "offset_points": offset_points,
+    }
+
+    #    requiredList = _default_remove_offset_parameters.keys()
```

Comment:
@tkeller12 Thanks for PR, please remove the commented lines to avoid causing future confusions.

Reply:
Okay. I thought I removed all these, but I missed a few. They are removed now.